ledger/eval/txntracer_test.go

```diff
@@ -264,13 +300,28 @@ int 1`,
 							},
 						},
 					},
+					{
+						Aidx: innerBoxAppID,
+						Addr: addrs[0],
+						Params: ledgercore.AppParamsDelta{
+							Params: &basics.AppParams{
+								ApprovalProgram: []byte{0x08, 0x26, 0x01, 0x0a, 0x67, 0x6f, 0x6f, 0x64, 0x62, 0x79, 0x65, 0x62, 0x6f, 0x78, 0x28, 0x81,
+									0x0a, 0xb9, 0x48, 0x28, 0x81, 0x00, 0x80, 0x01, 0x32, 0xbb, 0x28, 0xbc, 0x48, 0x81, 0x01},
+								ClearStateProgram: []byte{0x08, 0x81, 0x01},
+							},
+						},
+					},
 				},
 			}
 			expectedKvMods := map[string]ledgercore.KvValueDelta{
 				"bx:\x00\x00\x00\x00\x00\x00\x03\xe9hellobox": {
 					OldData: []uint8{0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
 					Data:    []uint8{0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
 				},
+				"bx:\x00\x00\x00\x00\x00\x00\x03\xefgoodbyebox": {
+					OldData: nil,
+					Data:    nil,
+				},
```

Comment:
So this ghost of boxes past is picked up because the box was created then deleted?